setup.py

```diff
@@ -51,33 +49,49 @@ def find_package_data(data_root, package_root):
 Operating System :: POSIX
 Operating System :: Unix
 Operating System :: MacOS
-Programming Language :: Python :: 2
-Programming Language :: Python :: 2.6
-Programming Language :: Python :: 2.7
 Programming Language :: Python :: 3
-Programming Language :: Python :: 3.3
-Programming Language :: Python :: 3.4
+Programming Language :: Python :: 3.7
+Programming Language :: Python :: 3.8
 """
 
 extensions = []
 
-setup(name='perses',
-      author='Patrick Grinaway',
-      author_email='patrick.grinaway@choderalab.org',
-      description=DOCLINES[0],
-      long_description="\n".join(DOCLINES[2:]),
-      version=__version__,
-      url='https://github.com/choderalab/perses',
-      platforms=['Linux', 'Mac OS-X', 'Unix'],
-      classifiers=CLASSIFIERS.splitlines(),
-      packages=['perses', 'perses.storage', 'perses.analysis', 'perses.samplers', 'perses.rjmc', 'perses.annihilation', 'perses.bias', 'perses.tests', 'perses.dispersed','perses.app','perses.utils'],
-      #package_data={'perses' : find_package_data('perses','examples') + find_package_data('perses','data')}, # I don't think this works
-      package_data={'perses' : find_package_data('perses/data', 'perses')}, # I think this is fixed
-      zip_safe=False,
-      entry_points={
-       'openmm.forcefielddir' : ['perses=perses:get_datadir'],
-       'console_scripts' : ['perses-relative = perses.app.setup_relative_calculation:run','perses-fah = perses.app.fah_generator:run']
-      },
-      ext_modules=extensions,
-      install_requires=[],
-      )
+setup(
+    name="perses",
+    author="Patrick Grinaway",
+    author_email="patrick.grinaway@choderalab.org",
```

Comment:
We should update the authorship lines here. I could not find [a way to specify multiple authors](https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py), so how about I claim responsibility for now?
```python
    author="John D. Chodera",
    author_email="john.chodera@choderalab.org",
```
```

Reply:
Sounds good! I view scientific software authorship like it works with papers, last name got the funding and is the person to email if you have questions!